include/bluetooth/services/gatts_client.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef BT_GATTS_CLIENT_H_
+#define BT_GATTS_CLIENT_H_
+
+/** @file
+ *
+ * @defgroup bt_gatts_client Bluetooth LE Generic Attribute Profile Service Client API
+ * @{
+ *
+ * @brief API for the Bluetooth LE Generic Attribute Profile (GATT) Service Client.
+ */
+
+#include <bluetooth/gatt.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt_dm.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Length of the Service Changed characteristic value. */
+#define BT_GATTS_SERVICE_CHANGED_VALUE_LENGTH 4
+
+/** @brief Structure for Service Changed handle range. */
+struct bt_gatts_handle_range {
+	/** Start handle. */
+	uint16_t start_handle;
+
+	/** End handle. */
+	uint16_t end_handle;
+};
+
+struct bt_gatts_client;
```

Comment:
We usually refer and use profile names instead when a profile is defined, for example `hogp`.
I'm thinking this might be better as `gattp`.

We don't have to separate between client and server in this case, since the server would always be in the stack itself. `_client` and `_c`  can be dropped from the names.

Reply:
Will do.